README.md

```diff
@@ -4,12 +4,19 @@
 Define the following: 
 
 - Recursion
+- When the result of a method dependes on the result of the same method but called with a smaller case. You know you have a recrusive method if the method calls its self 
 - Recursive Case
+- The expression of code where the method is calling its self
```

Comment:
"expression of code" is a weird term to me. Your idea is right, but seems like strange wording to me.